From 1e6aba6e0d4e1098b4e815db2e1d4dab9f2e7ffc Mon Sep 17 00:00:00 2001
From: Joerg Schambacher <joerg@hifiberry.com>
Date: Wed, 6 Oct 2021 17:19:58 +0200
Subject: [PATCH] Hifiberry DAC+ADCPro DT overlay: add optional headphone amp

This is a copy of the approach from our DAC+ driver.
It allows to probe (and activate) an optional TPA6130A2 headphone
amplifier.

Signed-off-by: Joerg Schambacher <joerg@hifiberry.com>
---
 .../boot/dts/overlays/hifiberry-dacplusadcpro-overlay.dts    | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/overlays/hifiberry-dacplusadcpro-overlay.dts b/arch/arm/boot/dts/overlays/hifiberry-dacplusadcpro-overlay.dts
index cafa2ccd7ff7..561cd84bbb79 100644
--- a/arch/arm/boot/dts/overlays/hifiberry-dacplusadcpro-overlay.dts
+++ b/arch/arm/boot/dts/overlays/hifiberry-dacplusadcpro-overlay.dts
@@ -43,6 +43,11 @@ hb_adc: pcm186x@4a {
 				clocks = <&dacpro_osc>;
 				status = "okay";
 			};
+			hpamp: hpamp@60 {
+				compatible = "ti,tpa6130a2";
+				reg = <0x60>;
+				status = "disabled";
+			};
 		};
 	};
 
-- 
2.32.0

